src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -12551,44 +12551,6 @@ protected struct S<T, U, V>
                 Diagnostic(ErrorCode.ERR_BadVisBound, "A<A<V>>").WithArguments("C2.S<T, U, V>", "C1.A<C1.A<V>>").WithLocation(12, 19));
         }
 
-        [Fact]
-        public void CS07ERR_BadVisBound02()
```

Comment:
>CS07ERR_BadVisBound02 [](start = 20, length = 21)

Why remove this test? It seems like the `ERR_BadVisBound` errors should still be reported.

Reply:
Restored the test.